src/nodes/harmony.ts

```diff
@@ -0,0 +1,15 @@
+// https://docs.harmony.one/home/developers/api#development-environments
+
+
+export const MAINNET_NODES: Array<string> = [ 
+	'https://api.harmony.one/',
+	'https://api.s0.t.hmny.io/',
+	'http://142.93.176.124:9500',
+	'http://167.71.160.56:9500',
+	'http://161.35.190.144:9500',
+	'http://209.97.145.163:9500',
```

Comment:
Remove all of them, we don't have S0 nodes and the rest are related to other Shards, so it would not work